tests/app/service/test_statistics.py

```diff
@@ -35,7 +35,9 @@
         StatsRow('email', 'technical-failure', 1),
         StatsRow('email', 'temporary-failure', 1),
         StatsRow('email', 'permanent-failure', 1),
-    ], [4, 0, 4], [0, 0, 0], [0, 0, 0]),
+        StatsRow('email', 'validation-failed', 1),
+        StatsRow('email', 'virus-scan-failed', 1),
+    ], [6, 0, 6], [0, 0, 0], [0, 0, 0]),
```

Comment:
then this line would have counts per channel:
```suggestion
    ], [4, 0, 4], [0, 0, 0], [1, 0, 1]),
```